### DISCUSSION TITLE
Timeschedule for version 6?

### DISCUSSION THREAD

**Discussion by Georg-Git** [tinymce/tinymce#7576] [General]
Is there already an idea when the first alpha release of Version 6 will be available?

Ugh - You already put a lot of stones in the way for migration.

Especially with a lot of renaming like:
fontsize_formats to
font_size_formats

lineheight_formats to
line_height_formats

As far as I can see the renamings are not connected with any changes of functionality giving the impression You do not want people to migrate to the version 6?
Really better to die in beauty?
😎

**Comment by TheSpyder** [comment]
We’ll enter internal QA for v6 in the next week or two. The community release is targeted for March; I’m not sure if we’re planning a public alpha or beta this time around.

There has been a wide scale effort to clean up inconsistencies in our API, and yes some of these do not involve feature changes. We will have a migration guide, there has been some discussion about building an automated migration tool but nothing is confirmed yet.
